keps/sig-storage/20190122-csi-inline-volumes.md

```diff
@@ -151,12 +151,20 @@ type CSIVolumeSource struct {
 ```
 
 ### Driver inline mode
-To indicate that a driver is setup to handle ephemeral inline volume requests, the existing `CSIDriver` object will be updated to include boolean attribute `InlineMode`.  When `CSIDriver.InlineMode == true` the followings are assumed:
+To indicate that the driver will support ephemeral inline volume requests, the existing `CSIDriver` object will be extended to include attribute `InlineMode`.  Currently the only InlineMode that will be supported is the `ephemeral` mode.  
```

Comment:
Sorry I think we should go back to the field from the original proposal that had both "ephemeral" and "persistent".  It makes it easier to add runtime validation when we make all the CSI calls.

Reply:
I tend to agree. I wasn't anticipating this much of a rework in response to Tim's suggestion to defer persistent inline volumes

Reply:
Yes, that's we'll do - https://github.com/kubernetes/enhancements/pull/790#discussion_r259579507

The rework is more design then code. The code will be simpler once we agree on the KEP.